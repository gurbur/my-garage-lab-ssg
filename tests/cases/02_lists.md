- 순서 없는 리스트 1번 항목
- 2번 항목
    - 중첩된 1번 항목
        - 더 깊게 중첩된 항목
    - 중첩된 2번 항목
- 3번 항목

1. 순서있는 리스트 1번
2. 2번 항목
    1. 중첩된 순서 있는 리스트
    2. 항목 2
        - 중첩된 순서 없는 리스트
3. 3번 항목
