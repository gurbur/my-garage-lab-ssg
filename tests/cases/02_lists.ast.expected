-> DOCUMENT
	-> UNORDERED_LIST
		-> LIST_ITEM
			-> TEXT | data1: " 순서 없는 리스트 1번 항목"
		-> LIST_ITEM
			-> TEXT | data1: " 2번 항목"
			-> UNORDERED_LIST
				-> LIST_ITEM
					-> TEXT | data1: " 중첩된 1번 항목"
					-> UNORDERED_LIST
						-> LIST_ITEM
							-> TEXT | data1: " 더 깊게 중첩된 항목"
						-> LIST_ITEM
							-> TEXT | data1: " 중첩된 2번 항목"
		-> LIST_ITEM
			-> TEXT | data1: " 3번 항목"
	-> ORDERED_LIST
		-> LIST_ITEM
			-> TEXT | data1: " 순서있는 리스트 1번"
		-> LIST_ITEM
			-> TEXT | data1: " 2번 항목"
			-> ORDERED_LIST
				-> LIST_ITEM
					-> TEXT | data1: " 중첩된 순서 있는 리스트"
				-> LIST_ITEM
					-> TEXT | data1: " 항목 2"
					-> UNORDERED_LIST
						-> LIST_ITEM
							-> TEXT | data1: " 중첩된 순서 없는 리스트"
		-> LIST_ITEM
			-> TEXT | data1: " 3번 항목"
